tests/unit/cn/cn_tree_test.c

```diff
@@ -785,15 +752,15 @@ cn_comp_work_init(
 
 MTF_DEFINE_UTEST_PRE(test, t_cn_comp, test_setup)
 {
-    enum cn_action action;
-    int            use_token;
-    int            cancel;
+    enum cn_action actions[3] = { CN_ACTION_SPILL, CN_ACTION_COMPACT_KV, CN_ACTION_COMPACT_K };
```

Comment:
static const please!

Reply:
Adding `const` does not change compiler output, which is not suprising.

Adding `static const` moves the structure out of the function, so it saves some run time and code space, but I don't see benefit to either in this case.

Reply:
const signals developer intent, static because it is function-scoped data. But up to you.

Reply:
Prob depends on how hot the function is, but it seems plausible the optimizer could just eliminate actions[] in some circumstances.  Damnit, now I have to go look at the code...

Reply:
it is a test file, so it doesn't matter regardless